site/docs/messaging/campaign-management/csp/campaign-api.mdx

```diff
@@ -379,6 +410,13 @@ Location: https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManag
 		<SubscriberHelp>true</SubscriberHelp>
 		<Sample1>test update sample</Sample1>
 		<AffiliateMarketing>false</AffiliateMarketing>
+		<MessageFlow>This is a sample Message Flow of my campaign.</MessageFlow>
+        <HelpKeywords>HELP</HelpKeywords>
+        <HelpMessage>This is a sample Help Message of my campaign.</HelpMessage>
+        <OptinKeywords>START, YES</OptinKeywords>
+        <OptinMessage>This is a sample Opt-In Message of my campaign.</OptinMessage>
+        <OptoutKeywords>STOP, NO</OptoutKeywords>
+        <OptoutMessage>This is a sample Opt-Out Message of my campaign.</OptoutMessage>
```

Comment:
Indentation?

Reply:
It just looks weird on the code. If you look at the response payload on http://bw-api-docs-mv-7602.s3-website-us-east-1.amazonaws.com/docs/messaging/campaign-management/csp/campaign-api/#request-url, it's correct. GH is behaving weird for some reason. When they were all aligned, it was looking weird on the actual doc